swarm-production/docker-stack.yml

```diff
@@ -89,6 +94,90 @@ services:
         delay: "10s"
         monitor: "5s"
 
+  queue:
+    image: 'wurstmeister/kafka:1.0.0'
```

Comment:
why not our kafka docker image?

Reply:
this one does auto-creation and auto-configuration while the scality one is vanilla (because configuration happens in the proprietary installer).